sauce/features/accounts/adjustable-column-widths/index.js

```diff
@@ -0,0 +1,157 @@
+import { Feature } from 'core/feature';
+import {
+  getCurrentRouteName,
+  getToolkitStorageKey,
+  removeToolkitStorageKey,
+  setToolkitStorageKey
+} from 'helpers/toolkit';
+
+// these must be in the order that they appear on the page
+const RESIZABLES = [
+  'ynab-grid-cell-date',
+  'ynab-grid-cell-accountName',
+  'ynab-grid-cell-payeeName',
+  'ynab-grid-cell-subCategoryName',
+  'ynab-grid-cell-memo',
+  'ynab-grid-cell-outflow',
+  'ynab-grid-cell-inflow',
+  'ynab-grid-cell-toolkit-running-balance'
+];
+
+export class AdjustableColumnWidths extends Feature {
+  injectCSS() { return require('./index.css'); }
+
+  constructor() {
+    super();
+    this.elementWasDragged = false;
+    this.isMouseDown = false;
+    this.currentX = null;
```

Comment:
This is just a style thing, but these can probably be initialised on the class instance like this:

```javascript
export class AdjustableColumnWidths extends Feature {
  elementWasDragged = false;
  isMouseDown = false;
  currentX = null;
}
```

I like not having constructors unless we need to do logic in them.

Of course this doesn't stop merging, feel free to tweak later if you agree.